drivers/sensor/si7006/si7006.c

```diff
@@ -115,25 +109,58 @@ static int si7006_channel_get(const struct device *dev,
 	struct si7006_data *si_data = dev->data;
 
 	if (chan == SENSOR_CHAN_AMBIENT_TEMP) {
```

Comment:
Might be worthwhile to convert to switch case?

Reply:
I would have used a case statement if I were writing it.  But it was already written by someone else and with only two possibilities the case statement isn't really any better.  I don't want to makes changes just to follow my style better, that just leads to too much code churn.  Everyone has an opinion.  So I'm just trying to change things to make them simpler or faster or fix bugs or add features, etc.